index.bs

```diff
@@ -5697,6 +5692,8 @@ At this time, one [=credential property=] is defined: the [=resident key credent
             If {{rk}} is [TRUE], the credential is a [=discoverable credential=];
             if {{rk}} is [FALSE], the credential is a [=server-side credential=].
             If {{rk}} is not present, it is not known whether the credential is a [=discoverable credential=] or a [=server-side credential=].
+
+            Note: some [=authenticators=] create [=discoverable credentials=] even when not requested by the user agent. Because of this, user agents may be forced to omit the {{rk}} property because they lack the assurance to be able to set it to [FALSE].
```

Comment:
The primary use case I have in mind is when the RP sets residentKey="preferred". If that happens, and the user agent first tries with rk=true, and that succeeds, it should always be the case that the user agent could return credProps with rk=true. I think it is important to distinguish that in the case where the user agent supports credProps but does not return rk within the credProps extension result, then it should be the case that the RP can infer the call from the user agent to the authenticator was made with rk=false (or omitted).

Reply:
That's a good point. I've updated this to note that user-agents should try to populate `rk`. Thus, if it's missing, the user-agent believes that the credential is non-discoverable — it's just can't be completely sure with CTAP 2.0.